packages/cozy-konnector-libs/docs/api.md

```diff
@@ -905,6 +906,38 @@ Get the data saved by saveAccountData
 Update account attributes and cache the account
 
 **Kind**: instance method of [<code>BaseKonnector</code>](#BaseKonnector)  
+<a name="BaseKonnector+waitForTwoFaCode"></a>
+
+### baseKonnector.waitForTwoFaCode()
+Tells Home that 2FA code is needed and wait for the user to submit it.
+It uses the account to do the communication with Home
```

Comment:
s/Home/Harvest

Reply:
Here the connector developer want to talk to the home application to let the user type the code. I think Harvest is an implementation detail at this level.